requirements_nvidia.txt

```
@@ -0,0 +1,14 @@
+--index-url https://resource.flagos.net/repository/flagos-pypi-hosted/simple
+--extra-index-url https://pypi.tuna.tsinghua.edu.cn/simple
+--trusted-host resource.flagos.net pypi.tuna.tsinghua.edu.cn
+torch>=2.2.0
+packaging
```

Please specify the version range as you have done for torch.
Pinning a version (or a version range) helps make sure the build result is reproducible.


```suggestion
packaging>=25.0
```